### ISSUE TITLE
Role not inside the required context

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9257]
# Issue

Role not inside the required context

This page element has a role that only makes sense in the context of another role.

https://my2.siteimprove.com/Inspector/998093/A11y/Page?pageId=24733541800&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r42/failed/RnRkMjdHNzk=;YWEyOGI1MGU=

May also be related to #9254
